### ISSUE TITLE
ScummVM cursor is shifted

### ISSUE THREAD

**Issue by legluondunet** [libretro-mirrors/scummvm#24]
Hello,

when I launch ScummVM core interface, the ScummVMcursor is shifted from my aim cursor, it is very annoying. How to have only one cursor?

**Comment by RobLoach** [comment]
Yeah, it's difficult to even play with the cursor like this. If you go full-screen, it's a bit more managable.

**Comment by NelsonRosenberg** [comment]
Any way to a least hide the mouse cursor like in the desmume core?

**Comment by NelsonRosenberg** [comment]
A good way to fix this is using the mouse focus mode of retroarch.

**Comment by RobLoach** [comment]
Using the Pointer API would really help the shifted mouse position: https://github.com/libretro/scummvm/issues/101
